modules/rosa-sdpolicy-security.adoc

```diff
@@ -0,0 +1,48 @@
+:system-module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-service-definition.adoc
+
+[id="rosa-sdpolicy-security_{context}"]
+= Security
+
+== Authentication provider
+Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] or cluster creation process or using the `rosa` CLI. The `rosa` CLI allows the use of `htpasswd`. {product-title} is not an identity provider, and all access to the cluster must be managed by the customer as part of their integrated solution. Multiple identity providers provisioned at the same time is supported. The following identity providers are supported:
+
+- GitHub or GitHub Enterprise
+- GitLab
+- Google
+- LDAP
+- OpenID connect
+
+== Privileged containers
+Privileged containers are not supported on {product-title}. To enable Red Hat to operate {product-title} as a managed service with an SLA, some restrictions are enforced to limit the ability of rogue or accidental changes that could impact the service.
+
+== Customer administrator user
+In addition to normal users, {product-title} provides access to an {product-title}-specific Group called `dedicated-admin`. Any users on the cluster that are members of the `dedicated-admin` group:
+
+- Have administrator access to all customer-created projects on the cluster
+- Can manage resource quotas and limits on the cluster
+- Can add and manage `NetworkPolicy` objects
+- Are able to view information about specific nodes and PVs in the cluster, including scheduler information
+- Can access the reserved `dedicated-admin` project on the cluster, which allows for the creation of Service Accounts with elevated privileges and also gives the ability to update default limits and quotas for projects on the cluster.
+
+== Cluster administration role
+The administrator of {product-title} has default access to the `cluster-admin` role for your organization's cluster. While logged into an account with the `cluster-admin` role, users have increased permissions to run privileged security contexts.
+
+== Project self-service
+All users, by default, have the ability to create, update, and delete their projects. This can be restricted if a member of the `dedicated-admin` group removes the self-provisioner role from authenticated users:
```

Comment:
s/All users, by default,/By default, all users
s/the self-provisioner role/the `self-provisioner` role

Reply:
Done.